src/session.rs

```diff
@@ -65,6 +65,31 @@ impl From<native_mux_impl::Session> for Session {
 // TODO: UserKnownHostsFile for custom known host fingerprint.
 
 impl Session {
+    /// Resume the connection using path to control socket and
+    /// path to ssh multiplex output log.
+    ///
+    /// If you do not use `-E` option (or redirection) to write
+    /// the log of the ssh multiplex master to the disk, you can
+    /// simply pass `None` to `master_log`.
+    ///
+    /// [`Session`] created this way will not be terminated on drop,
+    /// but can be forced terminated by [`Session::force_terminate`].
```

Comment:
This should be called `close` now I think.

Reply:
We should also update the documentation on `close` to mention that it will terminate the control master even if the session was resumed.

Reply:
Done!